app/assets/stylesheets/ltags/CtaTag.scss

```diff
@@ -0,0 +1,49 @@
+.ltag_cta {
```

Comment:
While this is meant to mimic site style, we're not going to re-use the utility classes, and instead we're mimicing other liquid tags and creating a new class `.ltag_cta`. This will give us more flexibility to define slight variations in different contexts as needed.